From 398d9ca0738c6395fd6863a4a04a8fd4ade9254c Mon Sep 17 00:00:00 2001
From: Manikanta Mylavarapu <quic_mmanikan@quicinc.com>
Date: Mon, 25 Sep 2023 19:09:27 +0530
Subject: [PATCH 202/500] scripts: Makefile: Add __QCOM_NON_SECURE_PIL__ to dtc
 flags

This change adds config to enable non-secure specific
DT properties.

Change-Id: Ic12b5cb0bc43ac366167248abe4f36df5a5b3836
Signed-off-by: Manikanta Mylavarapu <quic_mmanikan@quicinc.com>
---
 scripts/Makefile.lib | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/scripts/Makefile.lib b/scripts/Makefile.lib
index 9d318d33f416..be0b034a7a39 100644
--- a/scripts/Makefile.lib
+++ b/scripts/Makefile.lib
@@ -261,6 +261,10 @@ ifeq ($(CONFIG_CPU_THERMAL),y)
 dtc_cpp_flags += -D __CPU_THERMAL__
 endif
 
+ifeq ($(CONFIG_QCOM_NON_SECURE_PIL),y)
+dtc_cpp_flags += -D __QCOM_NON_SECURE_PIL__
+endif
+
 ifdef CONFIG_OBJTOOL
 
 objtool := $(objtree)/tools/objtool/objtool
-- 
2.34.1

